src/server.cpp

```diff
@@ -357,6 +359,7 @@ CServer::CServer ( const int          iNewMaxNumChan,
 
         // we always use stereo audio buffers (which is the worst case)
         vecvecsData[i].Init ( 2 /* stereo */ * DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES /* worst case buffer size */ );
+        vecvecsData2[i].Init ( 2 /* stereo */ * DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES /* worst case buffer size */ );
```

Comment:
Is this also needed if delayed panning is off? If not, do we need to initialize it (does it have any performance impact)? (also applies to line 345).

Reply:
Detlef Hennings is working on being able to operate the delay panning in different modes. This is about the distribution of the singers in the virtual space. 

Therefore it would be good to keep the parameter '-P panmode' free for this and similar developments and to use for delay panning only  --DelayPan (or similar) for delay panning.  

Furthermore, delay panning is so important for singers that it should be possible to activate it via a checkbox.

Reply:
> Furthermore, delay panning is so important for singers that it should be possible to activate it via a checkbox.

I think this is already part of this PR: https://github.com/jamulussoftware/jamulus/blob/0c6b3fbfc261b1b078bbd43ed939107deb07132e/src/serverdlgbase.ui

Reply:
> 
> 
> > Furthermore, delay panning is so important for singers that it should be possible to activate it via a checkbox.
> 
> I think this is already part of this PR: https://github.com/jamulussoftware/jamulus/blob/0c6b3fbfc261b1b078bbd43ed939107deb07132e/src/serverdlgbase.ui

Right now this is a server setting. I'd also prefer this to be a client setting but it isn't yet. If it is we wouldn't need the command line option.

Reply:
This would probably need a new protocol message.

Reply:
Possibly a misunderstanding: I did not mean that each client should have to start DelayPan individually. But the person who starts the server should have a checkbox to do that.

Reply:
> 
> 
> Possibly a misunderstanding: I did not mean that each client should have to start DelayPan individually. But the person who starts the server should have a checkbox to do that.

This checkbox is already there. At least in this version. But why not make this a client setting?

Reply:
Thought about it once more: it is a good idea after all to make this a client setting. Thanks for asking!

Reply:
There would need to be:
- a message from the server saying "I support delay pan if you want it"
- a message from the client saying "I would like delay pan"
- a client method to send the message
- a clientUI mechanism to request the feature _only if the server supports it_ (which would need to take server version and the specific message into account).  (I'm not sure if this should be in settings, as it's a user preference, or a on the server mixer, as the choice might vary depending on the jam session.)

I also think maybe get the server support finished and released "as is" before adding the client side.  I definitely think it's worth having, though.

Reply:
I‘d also like to get this done. The sooner the better. People would benefit immediately. The client part can come later and this should go out right away (3.7.0). But it is not on me to decide. I too can’t imagine anyone wanting to go back to volume panning. 

I‘d been entertaining the idea to provide a client setting in a later PR. The grounds are laid and there isn’t much to add to the server logic except the protocol bit. Mixing is per client already so we only need to make `bDelPan` a vector, similar to `fPan`.

Reply:
The problem is, because we want to be certain about the quality and fully understand operation of the code that goes into a release, every additional feature causes further delay in the release of 3.7.0, which already is later than we would have liked. That is not to express any doubt about the code quality or its usefulness, it just takes time.

It will definitely be fully reviewed for inclusion in 3.7.1, and hopefully that release will not be too far into the future.